examples/Demo/Shared/Pages/NavMenu/Examples/NavMenuCollapsible.razor

```diff
@@ -23,11 +34,12 @@
             Sed maximus, urna in fringilla posuere, enim urna bibendum justo, vel molestie nibh orci nec lectus.
             Etiam a varius justo. Aenean nisl ante, interdum eget vulputate eget, iaculis ut massa.
             Suspendisse maximus sed purus id molestie. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-        
+        </div>
     </div>
 </FluentStack>
 
 @code
 {
     bool Expanded = true;
-}
\ No newline at end of file
+    bool EnableCollapsedHierarchy = true;
```

Comment:
Maybe change name to `EnableCollapsedHierachyNavigation` to better convey what it does.

Reply:
I changed it to `EnableCollapsedChildNavigation` to better fit with the parameter name